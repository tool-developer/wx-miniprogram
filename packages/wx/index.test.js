import app from './index';

test('wx app test', () => {
  // 固定index.test.js.snap中该字段时间
  app.__SET_DATA_FREQUENCY_TIMESTAMP = 1625559145942;
  //
  expect(app).toMatchSnapshot();
});