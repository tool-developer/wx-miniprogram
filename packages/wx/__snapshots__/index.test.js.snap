// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`wx app test 1`] = `
Object {
  "SET_DATA_FREQUENCY": 100,
  "__SET_DATA_CACHE": Object {},
  "__SET_DATA_FREQUENCY_TIMESTAMP": 1625559145942,
  "data": Object {},
  "events": Object {},
  "extend": [Function],
  "version": "1.0.0",
}
`;
